planner/core/stats.go

```diff
@@ -1218,6 +1218,26 @@ func (p *LogicalCTE) DeriveStats(childStats []*property.StatsInfo, selfSchema *e
 
 	var err error
 	if p.cte.seedPartPhysicalPlan == nil {
+		// Build push-downed predicates.
+		if len(p.cte.pushDownPredicates) > 0 && p.cte.recursivePartLogicalPlan == nil {
+			selectedPushDownPredicates := make([]expression.Expression, 0)
+			minColSet := expression.IntersectColumnSet(p.cte.pushDownPredicates)
+			for _, cond := range p.cte.pushDownPredicates {
+				cnfItems := expression.SplitCNFItems(cond)
```

Comment:
The `pushDownPredicates` should already be in CNF form. Thus each of its items should be a DNF case. Why do we call `SplitCNFItems` here?